### PR TITLE
Add penguin class

### PR DESCRIPTION
**Pull request by BakerWJ**
I have made a penguin class! I made sure to keep our code super compact and neat because my hard drive is running out of space. The penguin is also named Rico.

### CODE REVIEW

**Review by EclipseIsDead — CHANGES_REQUESTED** [review]
This is really good stuff Baker Winston Jackson! I really like how  you implemented a name for the penguin, the children in my class are only differentiated by age. However, I noticed there's no documentation! This is bad practice!

**Comment by sugolov** [comment]
**Baker Winston Jackson**

**Comment by roryhofland** [comment]
rico

**Comment by kenneth-miura** [comment]
Dear Baker >_>,

I really enjoyed your code owo!!! (σ`・∀・)σ(σ`・∀・)σ(σ`・∀・)σ(σ`・∀・)σ

However, just one thing SON. I noticed you didn't do ANY documentaton uwu. Please fix this my good sir.

Sincerely,

Your Father,
Jefferson Baker

**Comment by lorena-b** [comment]
Good work

File: src/main/java/Penguin.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Penguin implements Tradable, Domesticatable {
+    /**
+     * Returns the price of the Penguin
+     *
```
Comment (lorena-b) [inline src/main/java/Penguin.java:4]: Very nice documentation

**Review by sugolov — APPROVED** [review]
well done baker, i like that you added documentation

File: src/main/java/Penguin.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    public String sound() {
+        return "MEEP";
+    }
+
+    /**
+     * Provides the name of a given Penguin
+     *
+     * @return the name of the Penguin
+     **/ public String getName() {
+        return "Rico";
```
Comment (kenneth-miura) [inline src/main/java/Penguin.java:27]: I agree that all penguins are named rico

**Review by kenneth-miura — APPROVED** [review]
Great job mr jefferson baker

**Review by lorena-b — APPROVED** [review]
Please merge this

**Review by EclipseIsDead — APPROVED** [review]

**Pull request merged**